templates/chain.html

```diff
@@ -138,39 +138,72 @@ <h4>Facts</h4>
         </div>
 
         <!-- adversary -->
-
+        
         <div id="adversary-profile" class="section-profile" style="display: none">
-          <div class="row">
-            <div class="column section-border" style="flex:25%;">
-              <img src="/chain/img/hacker.png">
-              <h4 style="margin-bottom:-15px">Adversaries</h4>
-              <p class="section-description">
-                  Adversaries are collections of ATT&CK TTPs, designed to test specific threats.
-                  Abilities with unmet requirements are faded out to show the adversary cannot use them unless an
-                  unlocking ability is added.
-              </p>
-              <select id="profile-existing-name" style="margin-top:-15px" onchange="loadAdversary();">
-                <option value="" disabled selected>Select an existing adversary</option>
-                {% for adv in adversaries %}
-                    <option value="{{ adv.id }}">{{ adv.name }}</option>
-                {% endfor %}}
-              </select>
-              <select id="advFactSource" onchange="refreshColorCodes()">
-                <option value="" selected>Overlay fact source</option>
-                    {% for s in sources %}
-                        <option value="{{ s.facts | map(attribute='property') | list }}">{{ s.name }}</option>
-                    {% endfor %}
-              </select>
-              <button id="advNewBtn" type="button" class="button-notready atomic-button" onclick="saveNewAdversary()">Save new</button>
-            </div>
-            <div id="phases" class="column adversary-header" style="flex:75%;text-align: left">
-                <input id="profile-goal" type="text" class="advGoal">
-                <input id="profile-description" type="text">
-                <br><br>
-                <div id="dummy"></div>
+            <div class="row">
+                <div class="column section-border" style="flex:25%;">
+                    <img src="/chain/img/hacker.png">
+                    <h4 style="margin-bottom:-15px">Adversaries</h4>
+                    <br>
+                    <div class="toggle">
+                        <label class="switch">
+                            <input type="checkbox" id="togBtnOp" checked onchange="toggleAdversaryView()">
```

Comment:
add some space above the toggle button (see the operation toggle button to compare)

Reply:
NP - yup was a mistake